charts/federation-v2/charts/webhook/templates/webhook.yaml

```diff
@@ -0,0 +1,58 @@
+---
+apiVersion: v1
+kind: Service
+metadata:
+  name: kubefed-webhook-server
+  namespace: {{ .Release.Namespace }}
+spec:
+  selector:
+    server: "true"
+  ports:
+  - port: 443
+    targetPort: 8443
+---
+{{- $ca := genCA "kubefed-webhook-ca" 3650 }}
+{{- $cn := printf "%s-webhook" .Release.Name }}
+{{- $altName1 := printf "kubefed-webhook-server.%s" .Release.Name .Release.Namespace }}
+{{- $altName2 := printf "kubefed-webhook.%s.svc" .Release.Name .Release.Namespace }}
+{{- $cert := genSignedCert $cn nil (list $altName1 $altName2) 3650 $ca }}
+apiVersion: admissionregistration.k8s.io/v1beta1
+kind: ValidatingWebhookConfiguration
+metadata:
+  name: federatedtypeconfigs.core.federation.k8s.io
+webhooks:
+- name: federatedtypeconfigs.core.federation.k8s.io
+  clientConfig:
+    service:
+      namespace: {{ .Release.Namespace }}
+      name: kubefed-webhook-server
+      path: /federatedtypeconfigs
+    caBundle: {{ b64enc $ca.Cert }}
+  rules:
+  - operations:
+    - CREATE
+    apiGroups:
+    - project.openshift.io
+    apiVersions:
+    - "*"
+    resources:
+    - projectrequests
+  - operations:
+    - CREATE
+    apiGroups:
+    - ""
+    apiVersions:
+    - "*"
+    resources:
+    - namespaces
+  failurePolicy: Fail
+---
+apiVersion: v1
+kind: Secret
+metadata:
+  namespace: openshift-namespace-reservation
```

Comment:
Shouldn't this also be the same namespace as that of the rest of the `webhook` resources.

Reply:
Yes, and it's updated in a subsequent commit.